### PR TITLE
Bump to 0.7.0

### PR DESCRIPTION
**Pull request by mira01**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: pyproject.toml [hunk 1]
```diff
@@ -89,11 +89,6 @@ src="gooddata-*-client/gooddata_*_client/configuration.py"
 # clients __init__.py
 src="gooddata-*-client/gooddata_*_client/__init__.py"
 
-[[tool.tbump.file]]
```
Comment (pcerny) [inline pyproject.toml:92]: Pleas bump version in:
- https://github.com/gooddata/gooddata-python-sdk/blob/master/.openapi-generator/configs/gooddata-afm-client.yaml#L9
- https://github.com/gooddata/gooddata-python-sdk/blob/master/.openapi-generator/configs/gooddata-metadata-client.yaml#L9
- https://github.com/gooddata/gooddata-python-sdk/blob/master/.openapi-generator/configs/gooddata-scan-client.yaml#L9

I forgot to update bump configuration when I did the change of `generate_client.sh` script.

**Review by pcerny — CHANGES_REQUESTED** [review]

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Review by pcerny — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-python-sdk-gate-PR65

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-sonarqube-scan-zuul-gate/58/ : SUCCESS in 51s
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-fossa-scanning-zuul-el7/58/ : SUCCESS in 2m 38s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-merge-compliance-check/58/ : SUCCESS in 19s

**Pull request merged**